modules/apps/portal-search/portal-search/src/main/java/com/liferay/portal/search/internal/spi/model/query/contributor/GroupIdQueryPreFilterContributor.java

```diff
@@ -52,9 +52,9 @@ public void contribute(
 		TermsFilter scopeGroupIdsTermsFilter = new TermsFilter(
 			Field.SCOPE_GROUP_ID);
 
-		for (int i = 0; i < groupIds.length; i++) {
-			long groupId = groupIds[i];
+		int currentIndex = 0;
 
+		for (long groupId : groupIds) {
```

Comment:
i think i would leave this a regular for loop unless there's a reason it has to be for-each?

Reply:
Yeah, Felipe - could you return this to the regular for loop, buy still keep the _addTermsFilters() method. I think the name "index" instead of "currentIndex".